ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/HD/LMDB.hs

```diff
@@ -252,42 +268,68 @@ initRangeReadLMDBTable count (LMDBMK _        db)                      = ApplyVa
 rangeReadLMDBTable     count (LMDBMK _        db) (ApplyKeysMK   keys) = ApplyValuesMK    <$> lmdbRangeReadTable count db keys
 
 {-------------------------------------------------------------------------------
- Db settings
+ Db state
```

Comment:
To discuss over a call, why did we change `settings` to `state`?

Reply:
On a second pass I think this makes sense, but I'd still like to confirm I understood it correctly: we're basically doing this because `settings` was a misnomer.